.github/workflows/main-push-trigger.yaml

```diff
@@ -0,0 +1,10 @@
+name: Main Push Trigger
```

Comment:
Don't we have a build job on main that we can already trigger from? Is the purpose of this trigger job to get things publishing faster?

Reply:
yeahp, may as well do the pages build in parallel